terraform/account-wide-infrastructure/README.md

```diff
@@ -15,64 +15,63 @@ Each subdirectory corresponds to each AWS account (`mgmt`, `prod`, `test` and `d
 
 ## Prerequisites
 
-- Initial AWS account [bootstrap](../bootstrap/README.md) completed
-- Required packages installed to enable running of `nrlf.sh` shell script see [Setup](../../README.md#setup)
-
-## Initialise shell environment
-
-To use `nrlf` shell script commands. We must initialise the shell environment. Ensure all packages are installed, run the following commands at the root of the repository.
-
-```shell
-poetry shell
-source nrlf.sh
-```
+Before deploying the NRLF account-wide infrastructure, you will need:
 
-This will enable the `nrlf` commands.
+- Initial AWS account [bootstrap](../bootstrap/README.md) completed
+- The required packages to build NRLF, see [the Setup section in README.md](../../README.md#setup).
 
 ## Deploy account wide resources
 
-Login to `mgmt` account as admin.
+To deploy the account wide resources, first login to the AWS mgmt account on the CLI.
+
+Then, initialise your terraform workspace with:
 
 ```shell
-nrlf aws login mgmt-adminyour
+$ cd ACCOUNT_NAME
+$ terraform init && ( \
+    terraform workspace new account_wide || \
+    terraform workspace select account_wide )
 ```
 
-Update account wide resources for accounts (e.g. `dev`, `test`, `prod`, `mgmt`)
+Replacing ACCOUNT_NAME with the name of your account, e.g `dev`, `test` etc.
 
-Initialise terraform
+Once you have your workspace, you can plan your changes with:
 
 ```shell
-nrlf terraform init <account> account_wide
+$ terraform plan \
+    -var 'assume_account=AWS_ACCOUNT_ID' \
+    -var 'assume_role=terraform'
```

Comment:
These aren't required for the management account I believe - that gets applied using your local role

Reply:
According to the TF config, it looks like it uses those to define the role to assume:

![image](https://github.com/NHSDigital/NRLF/assets/162149879/ccbfe61d-5403-4586-95dd-034eeed6553d)